README.md

```diff
@@ -37,7 +37,7 @@ See our [CONTRIBUTING.md](CONTRIBUTING.md).
 These steps describe a quick local setup.
 They are not suitable for most production use-cases because they keep all data inside containers.
 
-1. Store the following in the `docker-compose.yml` file:
+1. Create a folder `ferretdb` and store the following in the `docker-compose.yml` file in folder `ferretdb`:
```

Comment:
That's a good change, but I wonder if we can do even better. For example, can we define a network with a fixed name that does not include Docker Compose project name as a prefix? Or can we avoid using the project name if `docker exec ` command?